src/app/NoGrantsFound.tsx

```diff
@@ -0,0 +1,12 @@
+import NoGrants from 'pages/NoGrants';
+import { Route, Routes } from 'react-router';
+
+const NoGrantsFound = () => {
+    return (
+        <Routes>
+            <Route path="*" element={<NoGrants />} />
+        </Routes>
+    );
+};
```

Comment:
Making a stand alone routes object to keep users from being able to navigate out on accident.